### PR TITLE
AEA-714 / AEA-934: Prescription timestamps

### PR DESCRIPTION
**Pull request by mfjarvis**

### CODE REVIEW

File: tests/e2e/pact/broker/verify.ts [hunk 1]
```diff
@@ -4,14 +4,14 @@ let endpoint: string
 
 let token: string
 
-let sleepMs: number = 0
+let sleepMs = 0
```
Comment (conner-jowitt) [inline tests/e2e/pact/broker/verify.ts:7]: Why are we removing the type from here?

File: coordinator/src/services/translation/prescription/index.ts [hunk 2]
```diff
@@ -47,9 +48,9 @@ export function convertBundleToPrescription(fhirBundle: fhir.Bundle): prescripti
   hl7V3Prescription.author = convertAuthor(fhirBundle, fhirFirstMedicationRequest)
   hl7V3Prescription.responsibleParty = convertResponsibleParty(fhirBundle, fhirFirstMedicationRequest)
 
-  const validityPeriod = fhirFirstMedicationRequest.dispenseRequest.validityPeriod
-  const expectedSupplyDuration = fhirFirstMedicationRequest.dispenseRequest.expectedSupplyDuration
-  if (validityPeriod || expectedSupplyDuration) {
+  if (getCourseOfTherapyTypeCode(fhirMedicationRequests) === CourseOfTherapyTypeCode.CONTINUOUS_REPEAT_DISPENSING) {
```
Comment (booshibooshi) [inline coordinator/src/services/translation/prescription/index.ts:51]: 👍

File: coordinator/src/services/translation/prescription/practitioner.ts [hunk 3]
```diff
@@ -20,26 +21,38 @@ import {convertOrganizationAndProviderLicense} from "./organization"
 import {getProvenances} from "../common/getResourcesOfType"
 import * as errors from "../../../models/errors/processing-errors"
 import {identifyMessageType, MessageType} from "../../../routes/util"
+import moment from "moment"
 import {InvalidValueError} from "../../../models/errors/processing-errors"
 
 export function convertAuthor(
   fhirBundle: fhir.Bundle,
-  fhirFirstMedicationRequest: fhir.MedicationRequest,
-  convertPractitionerRoleFn = convertPractitionerRole
```
Comment (booshibooshi) [inline coordinator/src/services/translation/prescription/practitioner.ts:28]: 👍

File: tests/e2e/pact/broker/verify.ts (hunk 1 above)
Comment (mfjarvis) [reply tests/e2e/pact/broker/verify.ts:7]: eslint done it

Comment (mfjarvis) [reply tests/e2e/pact/broker/verify.ts:7]: Presumably because we weren't telling it anything more than it could already infer

**Review by booshibooshi — APPROVED** [review]

**Pull request merged**